templates/kubernetes-cluster.template

```diff
@@ -383,6 +383,9 @@ Resources:
               --stack '${AWS::StackName}' \
               --region '${AWS::Region}' \
               --resource K8sMasterInstance
+    CreationPolicy:
```

Comment:
Wow, I could have sworn that was in my last PR.  Maybe it got inadvertently dropped when I rebased.